### ISSUE TITLE
Issue with Multi-mob Patrols

### ISSUE THREAD

**Issue by ghost** [ccshiro/cc-buglist#1885]
If you engage a multi-mob patrol group, kill at least one of the members of said group and die, the rest of the members of that group will stop patrolling their pre-determined route. This occurred in Kil'sorrow Fortress and The Arcatraz.

![wowscrnshot_100215_150230](https://cloud.githubusercontent.com/assets/13790554/10261899/b83dc5a8-6964-11e5-9f64-97ff9dd8f7e3.jpg)
![wowscrnshot_100215_223535](https://cloud.githubusercontent.com/assets/13790554/10261901/ba941afa-6964-11e5-9eda-468a3b641f9a.jpg)

**Comment by KimmoKM** [comment]
It happened with Ghost Riders of Karabor as well. At the time, I actually thought it might be how it's supposed to work (because how would the respawn work? I didn't remember seeing the group waiting for their missing member to respawn, but I certainly didn't remember seeing stray riders either) and didn't report it as a result.

Now I checked in retail and at least currently the riders continued with one missing member who didn't respawn during the 45min I waited. Then, I killed the entire group and all four riders respawned after mere minutes.
